lua/caf/addons/server/lifesupport.lua

```diff
@@ -659,12 +659,10 @@ end
 
 function Ply:UpdateLSClient()
 	if self.environment then
-		local co2 = self.environment:GetO2Percentage() or -1

```

Comment:
Nope, this must be localy. Both of those getters have side effects. GetTemperature can actually kill you, even, which causes a problem in the netmsg.